spec/requests/categories_controller_spec.rb

```diff
@@ -1,6 +1,8 @@
 require 'rails_helper'
 
 describe CategoriesController do
+  let!(:admin) { Fabricate(:admin) }
```

Comment:
Do we need to create an admin for every single test? Using the bang version of `let` means we no longer lazy evaluate the block.